src/agoradatatools/process.py

```diff
@@ -123,7 +126,14 @@ def process_dataset(
             staging_path=staging_path,
             filename=dataset_name + "." + dataset_obj[dataset_name]["final_format"],
         )
+    elif isinstance(df, list):
+        json_path = load.list_to_json(
+            df=df,
+            staging_path=staging_path,
+            filename=dataset_name + "." + dataset_obj[dataset_name]["final_format"],
+        )
     else:
+        # If it's a type that we don't support, we will process it as a DataFrame
```

Comment:
This comment isn't accurate... DataFrames will fall through to this else statement and we support them. I think it's fine to just remove the comment.

Reply:
Very true! Comment has been removed.